<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zuhe</class>
 <widget class="QWidget" name="zuhe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房贷计算器</string>
  </property>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>17</y>
     <width>681</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>贷款类别：   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>组合型贷款</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>商业贷款</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>公积金贷款</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>480</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>商业贷款额： </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>万</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>商贷利率：    </string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>420</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>公积金贷款额：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>万</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>公积金利率：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>420</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>按揭年数：    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>25年（300期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24年（288期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23年（276期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22年（264期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21年（252期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20年（240期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19年（228期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18年（216期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17年（204期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16年（192期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15年（180期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14年（168期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13年（156期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12年（144期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11年（132期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10年（120期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9年（108期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8年（96期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7年（84期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6年（72期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5年（60期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4年（48期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3年（36期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2年（24期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1年（12期）</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>480</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>还款方式：    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>等额本息</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>等额本金</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_8" native="true">
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>开始计算</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>0</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>清空重填</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_14" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="3">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_13" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_12" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_16" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_15" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>230</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
